tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -157,21 +157,26 @@ sub run {
         $reg_mode = 'suseconnect';
     }
     my $ansible_playbooks;
+    my %playbook_configs = (
+        ha_enabled => $ha_enabled,
+        registration => $reg_mode,
+        fencing => get_var('FENCING_MECHANISM'));
+
     if (get_var('PTF_ACCOUNT') && get_var('PTF_CONTAINER') && get_var('PTF_KEYNAME')) {
-        $ansible_playbooks = create_playbook_section_list(
```

Comment:
here there was two calls to  `create_playbook_section_list` only to provide a different set of arguments.
Change the code to only have one call and only use if/else to  compose arguments, provided as a unique hash